lib/errors.js

```diff
@@ -10,6 +10,8 @@ const chalk = require('chalk');
  */
 class CliError extends Error {
     constructor(options) {
+        const originalError = {};
```

Comment:
This is not a const.

Reply:
Hmmm... weird... Because `lint` tells me different:

```
yarn run v1.3.2
$ eslint bin lib test extensions

/Users/aileen/code/Ghost-CLI/lib/errors.js
  13:13  error  'originalError' is never reassigned. Use 'const' instead  prefer-const

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
```

Reply:
this would be a const, yes. Because it's being assigned to an object, you can mutate that object's properties regardless of whether or not it's a const.

Reply:
I didn't knew THAT 😁